.gitignore

```diff
@@ -97,3 +97,5 @@ gradle-app.setting
 # # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
 # gradle/wrapper/gradle-wrapper.properties
 
+
+application.properties
```

Comment:
Może dodawanie tego do .gitignore to lekka przesada? Nie jestem pewien jakie są "dobre praktyki" w tej kwestii, ale u nas w tym pliku siedzi konfiguracja bazy i wolałbym, żeby siedziała w repo.